crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/bootstrap.rs

```diff
@@ -179,16 +179,20 @@ pub async fn proxy_list(url: Url2) -> KitsuneP2pResult<Vec<Url2>> {
 
 #[cfg(test)]
 mod tests {
+    // Leaving these tests that no longer compile in so that they may be re-written in the future.
+    // Rather than directly testing against the cloudflare service, we should spin up a local rust
+    // bootstrap instance to test against.
+    /*
     use super::*;
     use crate::fixt::*;
     use crate::types::KitsuneAgent;
     use crate::types::KitsuneBinType;
     use crate::types::KitsuneSignature;
     use ::fixt::prelude::*;
-    use kitsune_p2p_types::dependencies::lair_keystore_api_0_0::internal::sign_ed25519::sign_ed25519_keypair_new_from_entropy;
     use kitsune_p2p_types::KitsuneError;
     use std::convert::TryInto;
     use std::sync::Arc;
+    */
 
     // TODO - FIXME - davidb
     // I'm disabling all these tests that depend on outside systems
```

Comment:
If you were to get these tests compiling, even if they fail and are ignored, that would help prevent further rot over time and increase the likelihood that these tests will actually get reinstated later. You could use `todo!()` to fill in any gaps. If the effort is not too great, and if there is more real usable code than blanks to fill in, I would suggest that as part of this PR.

Reply:
@maackle - added stubs for missing apis